db/schema.rb

```diff
@@ -267,6 +268,7 @@
     t.datetime "deleted_at"
     t.bigint "moderator_group_id"
     t.boolean "smtp_enabled", default: false
+    t.datetime "last_received_at"
```

Comment:
This is not related to this PR. Could you remove it? And check if the migration is valid? Run the following:
1. `git checkout staging`
1. `rails db:reset`
1. `git checkout <branch_name>`
1. `rails db:migrate`

This checks if your migrations works from staging as well

Reply:
Yup that was fixed by merging staging into the branch.
I've ran those commands and it migrated successfully